src/libcore/marker.rs

```diff
@@ -38,10 +53,11 @@ impl<T: ?Sized> !Send for *const T { }
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T: ?Sized> !Send for *mut T { }
 
-/// Types with a constant size known at compile-time.
+/// Types with a constant size known at compile time.
 ///
-/// All type parameters which can be bounded have an implicit bound of `Sized`. The special syntax
-/// `?Sized` can be used to remove this bound if it is not appropriate.
+/// All type parameters which can be bounded have an implicit bound of `Sized`.
```

Comment:
I'm not sure. The only case I know of where the `Sized` bound isn't automatic is the implicit `Self` parameter of a trait definition.